oneflow/api/python/framework/layout.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include <pybind11/pybind11.h>
+#include <pybind11/operators.h>
+#include "oneflow/api/python/of_api_registry.h"
+#include "oneflow/api/python/framework/tensortype.h"
+#include "oneflow/api/python/functional/common.h"
+#include "oneflow/core/framework/dtype.h"
+#include "oneflow/core/framework/layout.h"
+
+namespace py = pybind11;
+
+namespace oneflow {
+
+ONEFLOW_API_PYBIND11_MODULE("", m) {
+  py::class_<Symbol<Layout>, std::shared_ptr<Symbol<Layout>>>(m, "layout")
```

Comment:
```suggestion
  py::class_<Symbol<Layout>>(m, "layout")
```
看是不是直接这样就可以，shared_ptr 套 Symbol 也是不必要的